public/app/core/components/AppChrome/TopBar/ProfileButton.tsx

```diff
@@ -0,0 +1,66 @@
+import { css } from '@emotion/css';
+import { cloneDeep } from 'lodash';
+import { useToggle } from 'react-use';
+
+import { GrafanaTheme2, NavModelItem } from '@grafana/data';
+import { Dropdown, Menu, MenuItem, ToolbarButton, useStyles2 } from '@grafana/ui';
+import { contextSrv } from 'app/core/core';
+import { t } from 'app/core/internationalization';
+
+import { enrichWithInteractionTracking } from '../MegaMenu/utils';
+import { NewsContainer } from '../News/NewsDrawer';
+
+import { TopNavBarMenu } from './TopNavBarMenu';
+
+export interface Props {
+  profileNode: NavModelItem;
+}
+
+export function ProfileButton({ profileNode }: Props) {
+  const styles = useStyles2(getStyles);
+  const node = enrichWithInteractionTracking(cloneDeep(profileNode), false);
+  const [showNewsDrawer, onToggleShowNewsDrawer] = useToggle(false);
+
+  if (!node) {
+    return null;
+  }
+
+  const renderMenu = () => (
+    <TopNavBarMenu node={profileNode}>
+      <Menu.Divider />
+      <MenuItem
+        icon="rss"
+        onClick={onToggleShowNewsDrawer}
+        label={t('navigation.rss-button', 'Latest from the blog')}
+      />
```

Comment:
this is missing the `config.newsFeedEnabled` check

Reply:
resolved